src/test/java/api/support/fixtures/DeclareLostFixtures.java

```diff
@@ -42,6 +42,7 @@ public Response declareItemLost(JsonObject loanJson) {
     final DeclareItemLostRequestBuilder builder = new DeclareItemLostRequestBuilder()
       .forLoanId(loanId)
       .on(DateTime.now(DateTimeZone.UTC))
+      .withServicePointId(UUID.randomUUID())
```

Comment:
Correct.  This specific code is only used for a few tests, and in the context of those tests, wether the servicePointId actually belongs to a real servicepoint does not seem to matter-they're testing other aspects of declare lost behavior.

Reply:
> Correct. This specific code is only used for a few tests, and in the context of those tests, wether the servicePointId actually belongs to a real servicepoint does not seem to matter-they're testing other aspects of declare lost behavior.

Is the intent to demonstrate that it's ok to do this or does it just happen to work?

If the intent is to demonstrate the general use of the system, I think it would be preferable to provide an actual service point here.

 ```suggestion
      .withServicePointId(servicePointsFixture.cd1().getId())
```

Reply:
This won't compile.  I tried something very similar when I first looked at this code.  In order to be used, servicePointsFixture has to be initialized with a ResourceClient.  The actual tests that use this probably set it up in the setup code. However, that class and its resourceclient are not visible from here, so you'd either have to create one or pass it in if you want to use it that way here.  Creating your own resourceclient here is also going to require supplying all the data structures it requires to initialize, and so on.  This seemed like a lot of code to support one function, so I looked at the tests that were using it, saw that the servicepoint data did not seem to be integral to the test, and provided dummy data instead.  As I read it, this is a convenience function for tests that just need a declared lost loan-the specific settings don't particularly matter.  If you feel strongly about this, then it might be better to assess wether or not we should have such a convenience function-perhaps tests should just be expected to create service points when they want to construct declare lost requests, in which case they should just use the other methods in this fixture that expect that data.

Reply:
Fair enough, maybe put a comment explaining the reason for the use of a dummy service point ID so folks can understand it later.

> f you feel strongly about this, then it might be better to assess wether or not we should have such a convenience function-perhaps tests should just be expected to create service points when they want to construct declare lost requests

There are lots of things like this that I'd like to change / resolve, however now is likely not the time :-/

Reply:
>Fair enough, maybe put a comment explaining the reason for the use of a dummy service point ID so folks can >understand it later.

Doing that now.